wideq/core.py

```diff
@@ -199,7 +199,18 @@ def lgedm_post(url, data=None, access_token=None, session_id=None):
         headers['x-thinq-jsessionId'] = session_id
 
     with retry_session() as session:
-        res = session.post(url, json={DATA_ROOT: data}, headers=headers)
+        res = session.post(urljoin(api_root + '/', path),
+                           json={DATA_ROOT: data}, headers=headers)
+
+        if "rtiControl" in path:
```

Comment:
Let's move this from `lgedm_post`, which is a low-level communication primitive, to the places that actually use `rtiControl`. That way we won't have to check what kind of request we're doing; we'll do it in exactly the right place "by construction."

I think that might juts mean `set_device_controls`, or possibly that and `get_device_config`. Can we try it with just the former, and see if it works, and if not, do the latter?